nucliadb_node/src/settings.rs

```diff
@@ -258,6 +288,9 @@ pub struct EnvSettings {
     pub s3_indexing_bucket: String,
     pub s3_endpoint: Option<String>,
     pub azure_account_url: Option<String>,
+
+    // Mr.Flagly
+    pub flag_settings_url: Option<String>,
```

Comment:
note: we need to make sure this env var is propagated on `nucliadb_shared` or the node chart, otherwise the default flag will be rollout:100 (from the DEFAULT_FLAGS json)

Reply:
Yup, it's already done (via nucliadb-shared).

Something else I just realized I want to add, is to pass the environment to the FF check.